react-redux/.stylelintrc.json

```diff
@@ -6,5 +6,5 @@
     "scss/at-rule-no-unknown": true,
     "csstree/validator": true
   },
-  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+  "ignoreFiles": ["build/**", "public/**", "package.json", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
```

Comment:
Could you explain why this folder should be ignored? Is stylelint checking `package.json` file? 🤔 

Also, is there any file it should be checking in `public` folder as an example?

Reply:
Hello @bolah2009 !

I'm closing this pull request because I found the problems with the checkers are relative to my VIM configuration file, which I was using as a relative reference without adding the .css/.scss extensions.

I appreciate the attention and apologize for any misunderstanding!

Sincerely
Hefler